prow/cmd/deck/static/prow/prow.ts

```diff
@@ -649,42 +653,31 @@ function redraw(fz: FuzzySearch, pushState: boolean = true): void {
                 if (pulls.length) {
                     r.appendChild(cell.prRevision(`${org}/${repo}`, pulls[0]));
                 } else {
-                    r.appendChild(cell.text(""));
+                    r.appendChild(cell.text("-"));
                 }
             } else if (type === "batch") {
                 r.appendChild(batchRevisionCell(build));
             } else if (type === "postsubmit") {
                 r.appendChild(cell.commitRevision(`${org}/${repo}`, base_ref, base_sha, base_link));
             } else if (type === "periodic") {
-                r.appendChild(cell.text(""));
+                r.appendChild(cell.text("-"));
             }
         } else {
             // Don't render identical cells for the same PR/commit.
-            r.appendChild(cell.text(""));
-            r.appendChild(cell.text(""));
-        }
-        if (spyglass) {
-            // this logic exists for legacy jobs that are configured for gubernator compatibility
```

Comment:
.... I think you might also be losing support for legacy/gubernator jobs. But if the Results were linking to `buildUrl` (vs `gcsUrl`), it might be safe anyway. 🤔

Reply:
It has been added back!